1984bot.py

```diff
@@ -52,7 +52,79 @@
 @bot.event
 async def on_ready():
     print('ONLINE')
-
+    
+async def on_message(self, message):
+    if message.author == client.user:
+       return
+     string = message.content
+     stringArr = ['place', 'triggers', 'here']

```

Comment:
There's a blacklist file that gets parsed a bit earlier; I'll make that compatible here